rootfs/deploy.py

```diff
@@ -0,0 +1,48 @@
+import docker
+import os
+import tarfile
+import os.path
+import urllib
```

Comment:
I'd prefer if we used `python-requests` to `urllib` - https://gist.github.com/kennethreitz/973705 

Consistency across our code bases